.github/workflows/haskell-ci.yml

```diff
@@ -28,19 +28,34 @@ jobs:
     strategy:
       matrix:
         include:
+          - compiler: ghc-9.12.0.20241114
+            compilerKind: ghc
+            compilerVersion: 9.12.0.20241114
+            setup-method: ghcup
```

Comment:
this is due
```
ghcup-jobs: >=8.2
```

in `haskell-ci`. `haskell-ci` tries to use the default ghcup channel for 9.12 versions too.

Reply:
That configuration field should be removed. It's more harmful than useful.